xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-model/src/main/resources/xwiki.rest.model.xsd

```diff
@@ -618,4 +619,20 @@
       <element name="element" type="string" minOccurs="0" maxOccurs="unbounded"></element>
     </sequence>
   </complexType>
+
+  <complexType name="HierarchyItem">
+    <sequence>
+      <element name="label" type="string"></element>
+      <element name="name" type="string"></element>
+      <element name="type" type="string"></element>
+      <element name="entityReference" type="string"></element>
```

Comment:
``entityReference`` is not consistent with ``name`` and ``type``. Between ``entityName / entityType / entityReference`` and ``name / type / reference`` I would choose the later. But since you added ``name`` I would drop ``reference / entityReference`` for now. We can add it later if it's needed.